related to #6150: change GUI to something Locus-alike

eddiemuc opened the pull request:
related to #6150: change GUI to something Locus-alike
Will also fix #9327

For screenshots see https://github.com/cgeo/cgeo/issues/6150#issuecomment-720082544

fm-sys commented on main/res/values-cs/strings.xml:
```diff
@@ -777,7 +777,7 @@
     <string name="map_source_osm_offline">Offline</string>
     <string name="map_source_osm_offline_combined">Kombinované (Offline)</string>
-    <string name="map_source_osm_offline_combined_attribution_details">Kombinovaná (%d map, %d atributů): klikněte pro podrobnosti</string>
+    <string name="map_source_osm_offline_combined_attribution_details">Kombinovaná (%d atributů): klikněte pro podrobnosti</string>
```
Your hard-coded translations will be overwritten by crowdin automatically. So if you want to provide these translations, you will have to translate them via the crowdin UI. Or, if you want to make it easy for yourself, you can simply ignore all the translations as outdated ones will be removed automatically...

eddiemuc replied:
Hi @fm-sys, thanks for the hint. I am aware of that, but I needed to make the modification because lint was complaining due to the mismatch of parameters in the strings vs parameters given in the code (because I reduced the number of paramus from 2 to 1).

So it is fine if those changes are overridden by Crowdin, but I need to make them in order to go through the Android Studio checks (and presumably also through the CI/Crowdin checks although I did not try this out)

bekuno replied:
Then it would be the best to delete such lines.

eddiemuc replied:
> Then it would be the best to delete such lines.

I still need the translation, it just does not contain 2 parameters an more (just one)

bekuno replied:
We only discuss on the translated strings, not for the english strings.
The translations are only interesting for your coding if it is necessary to use them in your tests.
Only the english strings are interesting otherwise. They are the fallback language if the string not exist in the used language.
The translation work what you have done is useless then. It will be replaced by the crowdin PR. The englich strings are the base for the work at crowdin, nothing else.

eddiemuc replied:
Strings are irrelevant with newest changes, so I guess discussion can be resolved

bekuno commented on main/res/values/strings.xml:
```diff
@@ -812,11 +812,9 @@
-    <string name="map_source_osm_offline_combined_attribution_pleasewait">Combined (%d maps, please wait…)</string>
-    <string name="map_source_osm_offline_combined_attribution_details">Combined (%d maps, %d attributions): click for details</string>
-    <string name="map_source_osm_offline_combined_attribution_single">Combined (%d maps): %s</string>
+    <string name="map_source_osm_offline_combined_attribution_details">%d attributions, click for details</string>
```
This is a `PluralsCandidate` for the lint check, see e.g.
https://ci.cgeo.org/job/cgeo%20pull%20request/3919/androidLintResult/package.-963969145/source.-6884541216426645145/#819
Please have a look at the discussion in https://github.com/cgeo/cgeo/issues/8456

eddiemuc replied:
> This is a `PluralsCandidate` for the lint check, see e.g.
> https://ci.cgeo.org/job/cgeo%20pull%20request/3919/androidLintResult/package.-963969145/source.-6884541216426645145/#819
> Please have a look at the discussion in https://github.com/cgeo/cgeo/issues/8456

This translation is used in code only if given parameter is 2 or more (never for singular case), see other parts of this PR. Thus the singular/plural problem does not apply here imho.

bekuno replied:
Please see at https://github.com/cgeo/cgeo/issues/8456#issuecomment-645583358.
`And the rules for those numbers seem to vary a lot from language to language, according to the docs we need to support zero, one, two, few, many and other, although most languages only use one or two of those identifiers.`
So we have to support two, few, many and other here.

eddiemuc replied:
Strings are irrelevant with newest changes, so I guess discussion can be resolved

eddiemuc commented:
Will also fix #9327

bekuno reviewed (commented):
Is it possible to use a copyright.xml instead the *.png?

eddiemuc commented:
@bekuno I assume this refers to #9088? 
I can't personally not provide an xml because I don't know how to produce it (keep in mind that we can't take the SVG as-is because I need it to have a white background which I added with GIMP). So is there anyone who can provide such an XML?

moving-bits commented:
@eddiemuc 
Sorry, I created a branch conflict for you by moving all distance related `TextView` fields into its own include `map_distance.xml`.

eddiemuc commented:
> @eddiemuc 
> Sorry, I created a branch conflict for you by moving all distance related `TextView` fields into its own include `map_distance.xml`.

No problem, will merge it

bekuno commented:
Yes, I meant the svg version. 
I hoped that the source for the new icons will present also the svg version.

eddiemuc commented:
> Yes, I meant the svg version.
> I hoped that the source for the new icons will present also the svg version.
> Where did you got the file from? I can try to reformat the file.

* Original source: https://material.io/resources/icons/?search=copyright&icon=copyright&style=baseline. Ths SVG however just provides the "black" part of the image, not the "white" filling in the middle and the "white ring" around it.
* Thus I opened the svg in gimp and added white filling as well as a white "circle" around it. Resulting gimp file:
[copyright.zip](https://github.com/cgeo/cgeo/files/5471318/copyright.zip)

bekuno commented:
Here you are. Reworked with inkscape and fine tuned with notepad++ (plus xml plugin). Then converted to xml with the AS vector asset studio.
[copyright.zip](https://github.com/cgeo/cgeo/files/5471613/copyright.zip)

eddiemuc commented:
> Here you are. Reworked with inkscape and fine tuned with notepad++ (plus xml plugin). Then converted to xml with the AS vector asset studio.
> [copyright.zip](https://github.com/cgeo/cgeo/files/5471613/copyright.zip)

@bekuno very cool, thanks a lot! I incorporated your copyright xml into the PR and deleted the PNGs

Lineflyer commented:
As the changes requested by @bekuno have been done and approval was signalled in #6150 lets merge this.
Thanks @eddiemuc for implementing and again taking the effort to optimize it further :)
